openid-connect-4-credential-issuance-1_0.md

```diff
@@ -494,56 +558,24 @@ In order to enable dynamic callbacks of the issuer to the end-user's wallet, the
 
 For non-normative examples of request and response, see section 11.6 in [@!OIDC4VP].
 
-Note: credential issuer's client_id with wallet
-Note: potentially add example with `wallet_issuer` and `user_hint` 
+Note to the editors: need to sort out credential issuer's client_id with wallet and potentially add example with `wallet_issuer` and `user_hint` 
 
 ### Successful Authorization Response
 
 Authentication Responses MUST be made as defined in Section 3.1.2.5 of [@!OpenID].
 
-Below is a non-normative example of
-```
-HTTP/1.1 302 Found
-  Location: https://wallet.example.org/cb?
-    code=SplxlOBeZQQYbYS6WxSbIA
-    &state=af0ifjsldkj
-```
-
 ### Authentication Error Response
 
 Authentication Error Response MUST be made as defined in section 3.1.2.6 of [@!OpenID].
 
-The following is a non-normative example of an unsuccessful token response.
-
-```json=
-HTTP/1.1 302 Found
-Location: https://client.example.net/cb?
-    error=invalid_request
-    &error_description=Unsupported%20response_type%20value
-    &state=af0ifjsldkj
-```
-
```

Comment:
I have initially included examples for the readers from the SSI space not that familiar with OIDC. I am ok removing these if it feels redundant.

Reply:
understood. I brought the examples back.